### DISCUSSION TITLE
Welcome to VECTR Discussions!

### DISCUSSION THREAD

**Discussion by thebleucheese** [SecurityRiskAdvisors/VECTR#147] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. 

Discussions are a great place for topics like how to use certain VECTR features or sharing a deployment configuration.

**Comment by kkamran1010** [comment]
can you see if I'm banned from posting in discussion tab? i do not see a way to post a discussion anymore, i have posted previously.

**Reply by SRAPSpencer** [reply]
We haven't banned anyone. We may have made changes to discussions permissions that had unintended changes, will look into posting permissions and update when assessed.

**Reply by kkamran1010** [reply]
awesome thanks!

**Reply by SRAPSpencer** [reply]
Should be resolved now, feel free to check.

**Reply by kkamran1010** [reply]
good to go!
